hyundai_kia_generic.dbc

```diff
@@ -792,7 +792,7 @@ BO_ 1362 SNV11: 4 SNV
  SG_ CF_Snv_IRLampControl : 10|2@1+ (1.0,0.0) [0.0|3.0] ""  BCM,CLU,HUD
 
 BO_ 593 MDPS12: 8 MDPS
- SG_ CR_Mdps_StrColTq : 0|11@1+ (0.0078125,-8.0) [-8.0|7.9921875] "Nm"  LDWS_LKAS
+ SG_ CR_Mdps_StrColTq : 0|11@1+ (1.0,-1024.0) [-1024.0|1024.0] ""  LDWS_LKAS
```

Comment:
Why change this? It's better to keep the DBC in real units if possible, unless the real units are wrong.